packages/editor/src/components/blocks/datatable/DataTable.tsx

```diff
@@ -43,17 +45,21 @@ export const DataTableComponent: ComponentConfig<DataTableProps> = {
     components: { subsection: 'Columns', label: 'Object-Bound Columns', type: 'selectColums' },
     paginator: { subsection: 'Paginator', label: 'Enable Paginator', type: 'checkbox' },
     maxRows: { subsection: 'Paginator', label: 'Rows per Page', type: 'number', hide: data => !data.paginator },
+    ...visibleComponentField,
     ...baseComponentFields
   },
   quickActions: ['DELETE', 'DUPLICATE', 'CREATECOLUMN']
 };
 
-const UiBlock = ({ id, components, value, paginator, maxRows }: UiComponentProps<DataTableProps>) => (
+const UiBlock = ({ id, components, value, paginator, maxRows, visible }: UiComponentProps<DataTableProps>) => (
   <>
     <div className='block-table'>
       <div className='block-table__label'>
         <span style={{ color: 'var(--N600)' }}>{value.length > 0 ? value : 'value is empty'}</span>
-        {paginator && maxRows !== '' && <span style={{ color: 'var(--N600)' }}>Rows per Page: {maxRows}</span>}
+        <Flex alignItems='center' gap={1}>
+          {paginator && maxRows !== '' && <span style={{ color: 'var(--N600)' }}>Rows per Page: {maxRows}</span>}
+          {UiBlockHeaderVisiblePart(visible)}
+        </Flex>
```

Comment:
Can't this here be done similar to the textarea info?  I mean the rows.

Reply:
Not quite, as the label is greyed out here. However, I have now introduced a flag where you can control whether the label should also be used as an indicator or not. In this way, I can also reuse the header component in the same way as the text area.